core/db/migrate/20211228133621_replace_orders_channel_column.rb

```diff
@@ -0,0 +1,6 @@
+class ReplaceOrdersChannelColumn < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :spree_orders, :channel
+    add_reference :spree_orders, :channel, foreign_key: { to_table: :spree_store_channels }
```

Comment:
@aleksandarpetrushev order should be the other way around

We also want to grab values from existing orders records and create new Channel records and associate them with existing orders, and only after that remove the channel column

Reply:
@damianlegawiec You mean Channel should have a FK to Order?

Reply:
@damianlegawiec